Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/nhinclib/NhincConstants.java

```diff
@@ -698,6 +698,40 @@ public static EVENT_LOGGING_SERVICE_NAME fromValueString(String valueString) {
     public static final String ADMIN_DASHBOARD_ERRORLOG_GETFILTERS = "getSearchFilter";
     public static final String ADMIN_DASHBOARD_ERRORLOG_VIEW = "viewErrorLog";
 
+    // load-test-data: ADMIN_LTD_SAVE/LIST/DELETE/GET
+    public static final String ENTITY_LOAD_TEST_DATA_SERVICE_NAME = "entityloadtestdata";
+    public static final String ADMIN_LTD_DELETEADDRESS = "deleteAddress";
+    public static final String ADMIN_LTD_DELETEIDENTIFIER = "deleteIdentifier";
```

Comment:
Can we move these constants into AdminGUI/AdminGUIWebservices project ..which ever project is more appropriate.